src/data/board.py

```diff
@@ -120,6 +128,12 @@ def permute_colors(self):
                 database.update_stored_pairing(self.black_pairing.stored_pairing)
             database.commit()
 
+    def update_board_result_from_pairing(self, pairing: 'Pairing', database: EventDatabase):
+        """Updates board timestamp. Clears board timestamp if result is NO_RESULT."""
+
+        if pairing.board:
+            pairing.board.stored_board.last_result_update = database.update_board_last_result_update(pairing.board.id, clear=pairing.result == Result.NO_RESULT)
```

Comment:
```suggestion
    def set_last_result_update(self, new_result: Result, database: EventDatabase):
        """Updates board timestamp. Clears board timestamp if result is NO_RESULT."""
        self.stored_board.last_result_update = database.update_board_last_result_update(
            self.identifier, clear=new_result == Result.NO_RESULT
        )
```
If that's a `Board` method, it should take effect on this board (i.e. self), not the one in the pairing.
It just needs the result as param, so let it just be that